src/callbacks/onPreprocess/imputeData/deriveVariables.js

```diff
@@ -0,0 +1,56 @@
+export default function deriveVariables() {
```

Comment:
A bit confused about new data flow - just need to go back and compare where this was happening before.